config/zones.json

```diff
@@ -115,6 +115,16 @@
       "production": "BG.fetch_production"
     }
   },
+  "BO": {
+    "capacity": {
+      "hydro": 483.22,
+      "unknown" : 1344.55,
+      "wind": 27 
+    },
+    "parsers": {
+      "production": "BO.fetch_production"
```

Comment:
Could not find it for DK. So I thought you were using it somewhere else maybe? 
Should I just add the line `"generation forecast": "BO.fectch_generation_forecast"` ?